Sources/Commands/SwiftTool.swift

```diff
@@ -677,6 +685,9 @@ public class SwiftTool<Options: ToolOptions> {
         if verbosity != .concise {
             args.append("-v")
         }
+        if let jobs = options.jobs {
+            args.append("-j(\(jobs))")
```

Comment:
Shouldn't this be `args.append("-j\(jobs)")`?

Reply:
Good catch. Is this code path ever used? I tested on Mac and Ubuntu, but I think they both used the libllbuild path.

Reply:
I guess not since we don't have `--disable-llbuild-library` flag. It might be time to get rid of `swift-build-tool` support :)